spinalcordtoolbox/download.py

```diff
@@ -39,8 +39,7 @@
     },
     "PAM50": {
         "mirrors": [
-            "https://github.com/spinalcordtoolbox/PAM50/releases/download/r20201104/PAM50-r20201104.zip",
-            "https://osf.io/download/5fa21326a5bb9d00610a5a21/",
+            "https://github.com/spinalcordtoolbox/PAM50/archive/refs/heads/master.zip",
```

Comment:
Note: I've changed the PAM50 link to use the files in the `master` branch, that way we can quickly test @valosekj's excellent contributions in https://github.com/spinalcordtoolbox/PAM50/pull/6 and https://github.com/spinalcordtoolbox/PAM50/pull/9. 

But, ultimately, this link should be replaced by a proper release of the PAM50 template. (Ideally, one that includes both the histology files _and_ the updated Phillips level files.)

Reply:
Thanks! 

Tested using the following commands and it seems to work well!

```console
# download PAM50 with histology atlas
git fetch
git checkout af/warp_histology_atlas
sct_download_data -d PAM50
# download testing data
sct_download_data -d sct_testing_data
# register t2 scan to PAM50
cd $SCT_DIR/data/sct_testing_data/t2
sct_register_to_template -i t2.nii.gz -s t2_seg-manual.nii.gz -lspinal t2_labeled_center_manual.nii.gz -c t2
# warp histology atlas
sct_warp_template -d t2.nii.gz -w warp_template2anat.nii.gz -histo 1 -a 0 -s 0
# check results
fsleyes t2.nii.gz label/histology/PAM50_200um_AVF.nii.gz
```

Reply:
This has finally been resolved -- a new release of the PAM50 template (including the histology files) has been created: https://github.com/spinalcordtoolbox/PAM50/releases/tag/r20230519

Unless there are any other concerns, I think this PR can be merged?